### PR TITLE
Fetch all the boards query by owner_id

### PR DESCRIPTION
**Pull request by jalapl**
Screenshot 2020-07-08 at 12 40 26

### Related issues
- This PR resolves #ISSUE_NUMBER

### Description

_Describe the changes and motivations for the pull request, unless obvious from the title._

### Todo

- [ ] _Insert todo_

### Review

- [ ] Check screenshots provided in the pull request
- [ ] Manual test №1 by _@replace_yourself_
- [ ] Manual test №2 by _@replace_yourself_

### Pre-merge checklist

- [ ] The PR relates to a single subject with a clear title and description in grammatically correct, complete sentences.
- [ ] Verify that feature branch is up-to-date with `master` (if not - rebase it).
- [ ] Double check the quality of [commit messages](http://chris.beams.io/posts/git-commit/).
- [ ] Squash related commits together.

### Screenshots

| Before                                        | After                                         |
| --------------------------------------------- | --------------------------------------------- |
| _Insert screenshots and/or screen recordings_ | _Insert screenshots and/or screen recordings_ |

| Screenshot < 1300px screen size               | Screenshot < 900px screen size                |
| --------------------------------------------- | --------------------------------------------- |
|            _Insert screenshot_                |             _Insert screenshot_               |

### CODE REVIEW

File: app/graphql/types/query_type.rb [hunk 1]
```diff
@@ -10,5 +10,14 @@ class QueryType < Types::BaseObject
...
     end
+
+    field :boards, [BoardType], null: true do
+      description "Find boards that belong to the owner"
+      argument :owner_id, ID, required: true
+    end
+
+    def boards(owner_id:)
+      Board.where(owner_id: owner_id)
+    end
```
Comment (akaoss) [inline app/graphql/types/query_type.rb:21]: And let's have this structure
```ruby
field :board
field :boards

def board
end

def boards
end
```

File: app/graphql/types/query_type.rb [hunk 2]
```diff
@@ -10,5 +10,14 @@ class QueryType < Types::BaseObject
     def board(id:)
       Board.find(id)
     end
+
+    field :boards, [BoardType], null: true do
+      description "Find boards that belong to the owner"
+      argument :owner_id, ID, required: true
+    end
```
Comment (akaoss) [inline app/graphql/types/query_type.rb:17]:
```suggestion
    field :boards, [BoardType], null: true do
      description "Find boards that belong to the group"
      argument :group_id, ID, required: true
    end
```

File: app/graphql/types/query_type.rb (hunk 1 above)
Comment (jalapl) [reply app/graphql/types/query_type.rb:21]: ok 👍

File: app/graphql/types/query_type.rb (hunk 2 above)
Comment (jalapl) [reply app/graphql/types/query_type.rb:17]: actually it works differently right now:
```ruby
Board.where('owner_id = ? OR group_id IN (?)', id, group_ids)
```

**Pull request merged**